tools/metaquantome/metaquantome_db.xml

```diff
@@ -0,0 +1,73 @@
+<tool id="db" name="metaQuantome: database" version="@GVERSION@">
```

Comment:
@caleb-easterly can we do the following here and change the ID to metaquantome_db ... similar to the other tools and change `   name_template: "metaquantome_{{ tool_id }}"` to `   name_template: "{{ tool_id }}"` this would make my life easier on usegalaxy.eu when I need to give you more memory. As I can specify the tool by metaquantome_db instead of db :)

Reply:
done!